### DISCUSSION TITLE
Which Basler/Pylon cameras are supported?

### DISCUSSION THREAD

**Discussion by robustcircuit** [bonsai-rx/bonsai#1247] [Q&A]
Hello,
I am about to build a few rigs for my lab and I am considering to use Bonsai to run most of the operations.
I am inclined to buy Basler cameras for their price/specifications balance but I see a few unanswered questions regarding Pylon nodes here and on the Google group. This makes me uncertain.
I am consider that specific one: 
https://www.baslerweb.com/en/products/cameras/area-scan-cameras/dart/daa1440-220uc-cs-mount/#specs
Would you recommend against using Basler/Pylon solution at all, or are there specific models supported and others to avoid? 
Best,
Romain

**Comment by chiyu1203** [comment]
for ace Basler camera, it works fine on Bonsai but not sure about that particular model

**Comment by bruno-f-cruz** [comment]
Bonsai doesnt target a specific camera, only the SDK. In this case, I would assume that anything compatible with the SDK version specified in the latest version of the package (pylon 7.1.0 runtime) should work.

**Reply by chiyu1203** [reply]
but then how come this [thread](https://github.com/orgs/bonsai-rx/discussions/2139) has some kind of weird problem @@

**Reply by bruno-f-cruz** [reply]
Not sure what the root cause of the error is, but as the stack points out, the exception is raised by their SDK, bonsai just propagates the error. Not to say that it can't be a problem on how the camera is being configured from
Bonsai or the external file. My point is that bonsai makes no prescription as to what cameras can be used, everything is handled by the sdk.

**Reply by slew000** [reply]
Hi Bruno, thanks a lot for pointing to the SDK problem. Do you have any suggestions on which parameters of the configuration file could be causing this issue? 
Also, do you have any ideas on why it sometimes works when I create a workflow from scratch, but it crashes if I save and load that same workflow?
